builds/podio-build.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+#---Create stampfile to enable our jenkins to purge old builds------------------------------
+touch $WORKSPACE/controlfile
+
+#---Set up environment----------------------------------------------------------------------
+source $WORKSPACE/podio/init.sh
+
+#---Prepare folder to store build artifacts-------------------------------------------------
+mkdir $WORKSPACE/podio/build
+cd $WORKSPACE/podio/build
+
+#---Run installation------------------------------------------------------------------------
+cmake -DCMAKE_INSTALL_PREFIX=../install -Dpodio_tests=ON ..
+make
```

Comment:
Could also speed up the build here by adding `-j 16`

Reply:
I think we can better use `make -j$(nproc)` so that the maximum number of processes depends on the build node, since we have an heterogeneous farm of nodes in terms of cores.